frontend/src/app/patients/EditPatient.tsx

```diff
@@ -69,7 +69,7 @@ interface GetPatientResponse {
     birthDate: string;
     street: string;
     streetTwo: string | null;
-    city: string | null;
+    city: string | "";
```

Comment:
isn't `""` a string?

Reply:
I believe so.  But we can still get back null from the database from older records and so we would want to default to empty string.  This defaulting to null is what was causing that gnarly error message you were seeing when editing an existing record without a city.

Reply:
yes, but this is just the type, right? and saying `city: string` would cover the case when `city` is `""`

Reply:
yeah it turns out I was misreading this syntax the whole time, the `| ""` is redundant / unnecessary